main.tf

```
@@ -17,7 +17,7 @@ module "tfe-workspace" {
   auto_apply                     = var.auto_apply
   auto_apply_run_trigger         = var.auto_apply_run_trigger
   branch                         = var.branch
-  clear_text_env_variables       = merge({ AWS_DEFAULT_REGION = var.region }, var.clear_text_env_variables)
+  clear_text_env_variables       = var.region != null ? merge({ AWS_DEFAULT_REGION = var.region }, var.clear_text_env_variables) : var.clear_text_env_variables
```

```suggestion
  clear_text_env_variables       = var.region != null ? merge(var.clear_text_env_variables, { AWS_DEFAULT_REGION = var.region }) : var.clear_text_env_variables
```

Should we swap these around to ensure setting the region via the variable takes precedence over it being a clear text env?